app/services/change_schedule.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+class ChangeSchedule
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations::Callbacks
+
+  attribute :cpd_lead_provider
+  attribute :participant_id
+  attribute :reason
```

Comment:
Could you remove `reason` as I don't think it's required for changing a schedule